src/components/common/HamburgerMenu.tsx

```diff
@@ -0,0 +1,12 @@
+const HamburguerMenu = () => {
+  return (
+    <img
+      loading="lazy"
+      src="src/assets/menu.svg"
```

Comment:
it's a good practice to import the src as an element
```suggestion

// top of the file: import { ReactComponent as Menu } from 'src/assets/menu.svg';
      src={Menu}
```